Automattic/jetpack
# Jetpack: Include Boost on "My Products"

Fixes Automattic/jpop-issues#7982

## Proposed changes:
* Include Boost on "My Products"

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Install and upgrade Boost on a test site
* Navigate to **Jetpack Dashboard > My Plan**
* Make sure boost is listed

![image](https://user-images.githubusercontent.com/3737780/212273463-b8b2b343-c927-4ab1-8bf9-2dcd2e800098.png)
